src/routes/__tests__/graphqlApolloPlugins.test.js

```diff
@@ -0,0 +1,58 @@
+import {
+    GraphQLObjectType,
+    GraphQLString,
+    GraphQLSchema,
+} from 'graphql';
+
+import { bind, setDefaults, getContainer, Nodule } from '@globality/nodule-config';
+
+jest.mock('apollo-server-express');
+
+import * as apolloServerExpress from 'apollo-server-express'; // eslint-disable-line import/first
+import '../graphql'; // eslint-disable-line import/first
+import '../../terminal'; // eslint-disable-line import/first
+
+const QueryType = new GraphQLObjectType({
+    name: 'QueryType',
+    description: 'Top-level queries',
+    fields: {
+        hello: {
+            type: GraphQLString,
+            resolve: () => 'world',
+        },
+    },
+});
+
+const schema = new GraphQLSchema({
+    query: QueryType,
+});
+
+bind('graphql.schema', () => schema);
+
+describe('routes.graphql', () => {
+
+    it('will supply apollo plugins configs to apollo server instance', async () => {
+        const mockApolloServer = jest.fn();
+        apolloServerExpress.ApolloServer.mockImplementation(mockApolloServer.mockReturnThis());
+
+        setDefaults('routes.graphql.apolloPlugins', {
+            requestDidStart() {
+                return {
+                    didEncounterErrors() {
+                        console.log('didEncounterErrors');
+                    },
+                };
+            },
+        });
```

Comment:
I think it's probably more idiomatic JS to structure like this:

```suggestion
        setDefaults('routes.graphql.apolloPlugins', {
            requestDidStart: () => ({
                didEncounterErrors: () => console.log('didEncounterErrors');
            }),
        });
```

We call `setDefaults` with an object that has a property called `requestDidStart` that's a function that returns an object. That object has a property called `didEncounterErrors` that's a function that happens to log to the console. If you don't want it to do that, you can do something like `() => null` (a function that just returns `null`).